test/sslapitest.c

```diff
@@ -2082,6 +2082,91 @@ static int test_psk_tickets(void)
 
     return testresult;
 }
+
+static int test_extra_tickets(int idx)
+{
+    SSL_CTX *sctx = NULL, *cctx = NULL;
+    SSL *serverssl = NULL, *clientssl = NULL;
+    int testresult = 0;
+    size_t nbytes;
+    unsigned char c, buf[1];
+
+    new_called = 0;
+    do_cache = 1;
+
+    if (!setup_ticket_test(0, idx, &sctx, &cctx))
```

Comment:
Should we also be testing this with stateful tickets (i.e. the first param here set to 1)?

Reply:
I do recall deciding to not test stateful tickets, but I don't remember why, so it clearly can't have been a good reason.
(added.)